Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedListItemView.swift

```diff
@@ -38,111 +38,124 @@ public struct PreplannedListItemView: View {
-                        } label: {
-                            Image(systemName: "arrow.down.circle")
-                        }
-                    case .failed:
-                        // An error occured when packaging the preplanned map area.
-                        Image(systemName: "exclamationmark.circle")
-                            .foregroundStyle(.red)
-                    }
+                        .lineLimit(2)
+                } else {
+                    Text("This area has no description.")
+                        .font(.footnote)
+                        .foregroundStyle(.tertiary)
                 }
+                statusView
             }
-            .onReceive(preplannedMapModel.preplannedMapArea.$loadStatus) { loadStatus in
-                let status = preplannedMapModel.preplannedMapArea.packagingStatus
-                
-                switch loadStatus {
-                case .loaded:
-                    // Allow downloading the map area when packaging is complete,
-                    // or when the packaging status is `nil` for compatibility with
-                    // legacy webmaps that have incomplete metadata.
-                    withAnimation(.easeIn) {
-                        packagingStatus = (status == .complete || status == nil) ? .complete : .packaging
-                    }
-                case .loading:
-                    if status == .processing {
-                        // Disable downloading map area when still packaging.
-                        packagingStatus = .packaging
-                    } else {
-                        packagingStatus = .notLoaded
-                    }
-                case .notLoaded:
-                    packagingStatus = .notLoaded
-                case .failed:
-                    // Disable downloading when the map fails to load.
-                    packagingStatus = .failed
+        }
+        .onReceive(preplannedMapModel.preplannedMapArea.$loadStatus) { loadStatus in
+            let packagingStatus = preplannedMapModel.preplannedMapArea.packagingStatus
+            
+            switch loadStatus {
+            case .loaded:
+                // Allow downloading the map area when packaging is complete,
+                // or when the packaging status is `nil` for compatibility with
+                // legacy webmaps that have incomplete metadata.
+                withAnimation(.easeIn) {
+                    areaStatus = (packagingStatus == .complete || packagingStatus == nil) ? .complete : .packaging
+                }
+            case .loading:
+                if packagingStatus == .processing {
+                    // Disable downloading map area when still packaging.
+                    areaStatus = .packaging
+                } else {
+                    areaStatus = .notLoaded
                 }
+            case .notLoaded:
+                areaStatus = .notLoaded
+            case .failed:
+                areaStatus = .failed
```

Comment:
```suggestion
                areaStatus = packagingStatus == .processing ? .packaging : .failed
```
The map area will fail to load while packaging. The state needs to be set to `.processing` here so the correct status text shows when packaging.

Reply:
this is fixed in the follow-up PR